### DISCUSSION TITLE
Enter key for message submit

### DISCUSSION THREAD

**Discussion by Ungaarr** [lencx/ChatGPT#1051] [Other]
Hi,
Is it known how to configure the program to submit messages by pressing the Enter key (without using the graphical Send button)?

**Comment by mofitn3577** [comment]
in the popup from the task menu you mean right? mine submits when I hit enter in the main window but in the popup which i use a lot you have to click it so i have been thinking about that issue and how to fix it also. I am assuming it could be fixed either with a custom script, or maybe by using powertools to assign a keyboard shortcut to the enter key maybe?

**Comment by Ungaarr** [comment]
from the window where I enter the text.
P.S. If I find an extension or program where it can be set or by default Enter will be submitted, I will not use this program.
